shims/spark320/src/main/scala/com/nvidia/spark/rapids/shims/spark320/Spark320Shims.scala

```diff
@@ -788,11 +790,12 @@ class Spark320Shims extends Spark32XShims {
   }
 
   override def getGpuShuffleExchangeExec(
-      outputPartitioning: Partitioning,
+      gpuOutputPartitioning: Partitioning,
```

Comment:
Can we have a follow on issue to separate out the GpuPartitioning from the Partitioning. Or explicitly mark it as GpuPartitioning in the method signature?

Reply:
Good catch, I meant to update the signature for `GpuPartitioning` and forgot to go back to update.  I'll at least do that in this PR and file a followup to separate them further to avoid accidentally passing a `GpuPartitioning` as a `Partitioning`.

Reply:
Filed #3733 as the followup.